### ISSUE TITLE
Don't cleanup quotation added by previous senders

### ISSUE THREAD

**Issue by rostafiychuk** [jeevatkm/ReplyWithHeaderMozilla#9]
It can be optional, but we shouldn't delete old quotation. We should delete added by Thunderbird during composing new email.

ReFwdFormatter does it better now.

**Comment by rostafiychuk** [comment]
Blockquotes can be fixed with:
```javascript
    handleBlockQuote: function() {
        ReplyWithHeader.Log.debug('handleBlockQuote()');
        let blockquotes = this.byTagName('blockquote');
        if (blockquotes.length>0) {
          blockquotes[0].setAttribute('style', this.bqStyleStr);
        }
    },
```

**Comment by presidentio** [comment]
This functionality will be useful for me too

**Comment by jeevatkm** [comment]
I will make it optional and user will have an options in the preference to choose. BTW are you interested in sending in PR for this?

**Comment by rostafiychuk** [comment]
Please review my pull request.

**Comment by jeevatkm** [comment]
It is merged and I have share download link in other issue you have reported to collect more info. Thanks.

**Comment by jeevatkm** [comment]
PR merged, will be release as part of v1.4 :)
